### PR TITLE
Species list integration

### PR DESCRIPTION
**Pull request by djtfmartin**
Contains integration with ALA's species lists (https://lists.ala.org.au) to retrieve and index occurrence records with:

* Links to species lists 
* Conservation status at stateProvince and country level
* Invasive status at stateProvince and country level

See https://github.com/AtlasOfLivingAustralia/la-pipelines/issues/3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ livingatlas/debian/*.substvars
 livingatlas/debian/files
 livingatlas/debian/la-pipelines
 livingatlas/debian/maven-repo-local/
+/livingatlas/pipelines/src/test/resources/image-service-non/dr893/image.csv-sorted
```
Comment (charvolant) [inline .gitignore:26]: This looks like an accidental inclusion?

File: livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java [hunk 2]
```diff
@@ -51,6 +51,25 @@ public void set() {
     }
   }
 
+  @Test
+  public void testGeocodeStae() {
```
Comment (charvolant) [inline livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java:55]: testGeocodeDate() ?

**Review by charvolant — COMMENTED** [review]
Two, I think, minor typos

**Comment by charvolant** [comment]
Conflicts probably caused by late merge of image service PR

File: livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java (hunk 2 above)
Comment (djtfmartin) [reply livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java]: removed.

File: .gitignore (hunk 1 above)
Comment (djtfmartin) [reply .gitignore]: removed.

**Pull request merged**